lib/rules/semi-style.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * @fileoverview Rule to enforce location of semicolons.
+ * @author Toru Nagashima
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const astUtils = require("../ast-utils");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+const SELECTOR = `:matches(${
+    [
+        "BreakStatement", "ContinueStatement", "DebuggerStatement",
+        "EmptyStatement", "ExportAllDeclaration", "ExportDefaultDeclaration",
+        "ExportNamedDeclaration", "ExpressionStatement", "ImportDeclaration",
+        "ReturnStatement", "ThrowStatement", "VariableDeclaration"
+    ].join(",")
```

Comment:
I was using `:statement` at first but I switched it to this because I need to ignore `IfStatement`, `ForStatement`, `ForInStatement`, `ForOfStatement` and `WhileStatement` in order to avoid checking in duplicate.

Is it better if I use `:statement` and `if` statement in the handler?